graphbolt/include/graphbolt/fused_sampled_subgraph.h

```diff
@@ -54,23 +54,28 @@ struct FusedSampledSubgraph : torch::CustomClassHolder {
    */
   FusedSampledSubgraph(
```

Comment:
Can you update the doc string of each field according to our new agreement such that we don't have ambiguous of the fields in the class.